frank/geometry.py

```diff
@@ -24,8 +24,6 @@
 
 import numpy as np
 from scipy.optimize import least_squares
-#import logging
```

Comment:
This isn't the way to handle a conversation. I'll now need to create a new branch with the logging changes. It would be better to just not merge this branch until we discuss further. If you run the code with the logging inline, you'll see it works as expected. So I'll make a new branch, but in the future please don't do this.

Reply:
In future, one thing per PR